bin/cloudgov/deploy-cron

```diff
@@ -116,8 +116,8 @@ if existsCFApp ${APPNAME} ; then
   cf_push $APPSPEC $APPINSTANCES
 else
   cf_push $APPSPEC $APPINSTANCES
-  $SCRIPT_DIR/setup-egress-for-apps ${APPNAME}
-  if [ $retval != 0 ]; then
+  $SCRIPT_DIR/setup-egress-for-apps ${APPNAME} --no-restart
+  if [ $retval != 0 ]; then ## Should retval be set above here?
```

Comment:
Yes, retval should be set --  just like on line 111. 
I'll bet we just haven't noticed because we keep re-deploying and we don't actually hit this "else" clause.